src/Symfony/Component/VarDumper/Cloner/VarCloner.php

```diff
@@ -128,6 +128,15 @@ protected function doClone(mixed $var): array
                             $stub->value = $v;
                             $stub->handle = $h;
                             $a = $this->castObject($stub, 0 < $i);
+                            if ($stub->value !== '') {
```

Comment:
Does this work for nested objects or should we implement that logic in `castObject` ?

And I'm wondering whether this should be implemented in `Caster::castObject` by adding those properties as a special `UninitializedStub`, so that this is done _before_ any custom caster runs. This way, if a caster decides to cut a property, this would also cut it if it is uninitialized.

Reply:
Ok I'll try implementing this solution. Thanks